concordium-std-derive/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "concordium-std-derive"
+authors = ["Concordium <developers@concordium.com>"]
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.65"
+license = "MPL-2.0"
+description = "Procedural macros to generate concordium type constants from strings, that makes writing smart contracts on the Concordium blockchain easier."
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+proc-macro = true
+
+[dependencies]
+# Replace with new version, obviously...
```

Comment:
Todo

Reply:
You don't have to use the `git` dependency. The contracts-common is part of concordium-base, which is a submodule of this repository. Please use taht dependency.